build/Tests/UnitTests/AudioToolbox/AudioToolbox.UnitTests.vcxproj

```diff
@@ -135,7 +135,7 @@
     <Link>
       <SubSystem>Console</SubSystem>
       <GenerateDebugInformation>true</GenerateDebugInformation>
-      <AdditionalDependencies>;libdispatch.lib;mincore.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>AudioToolbox.lib;mfuuid.lib;mfplat.lib;libdispatch.lib;mincore.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
@afaruqui I cannot in good faith just re-test the current build as I see the issue and agree with the build error. Any successful builds are likely due to chance and the binary in question just happening to be built in time but without a strong ordering enforced by the build machine. If I were to accept as-is, even with testing again, I would be knowingly making our build results unstable going forward.

All of that being said, it should be pretty easy to make the correct fix here. You mention unresolved external symbols for  AudioConverterNew  and AudioConverterDispose. Those symbols are found in **AudioConverter.mm** and declared in **AudioConverter.h**.  You can see that AudioToolboxLib.vcxproj is responsible for building this .mm file which means that the project reference should be sufficient to pull in the lib that has the implementations in question. 

This then begs the question why is the linker complaining about unresovled **external** symbols if they are in a lib being built with the unit tests. Well the key thing there is that they shouldn't be external. They are located simply in another TU in that module. It appears that changing the preprocessorDefines as mentioned in a comment I"m about to make should do the trick.